fe/fe-core/src/main/java/org/apache/doris/nereids/parser/LogicalPlanBuilder.java

```diff
@@ -435,13 +438,13 @@ public Map<String, String> visitPropertyClause(PropertyClauseContext ctx) {
 
     @Override
     public Map<String, String> visitPropertyItemList(PropertyItemListContext ctx) {
-        Builder<String, String> propertiesMap = ImmutableMap.builder();
+        Map<String, String> propertiesMap = Maps.newHashMap();
         for (PropertyItemContext argument : ctx.properties) {
             String key = parsePropertyKey(argument.key);
             String value = parsePropertyValue(argument.value);
             propertiesMap.put(key, value);
         }
-        return propertiesMap.build();
+        return propertiesMap;
```

Comment:
must use immutable map, do not change this

Reply:
done